docs/product-user-guide/src/main/asciidoc/business-central-settings-creating-new-users-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='business-central-settings-creating-new-users-proc']
+= Creating new users
+You can create as many {PRODUCT} users as you require. User privileges and settings are controlled by the roles assigned to a user and the groups that a user belongs to.
+
+.Procedure
+. Click the gear icon, then click *Users*.
+. Click *New user*, enter a user name, then click *Next*.
+. If desired, click the *Groups* tab, click *Add to groups*, select the groups that you want this user to belong to, then click *Add to selected groups*.
```

Comment:
Yeah, I see what you mean. What I do with things like this is to make two sentences, with 3 actions or less per sentence, or whatever seems logical. Still feels good in a single step, but makes it feel less packed.
So like this:
"If desired, click the *Groups* tab, click *Add to groups*, and select the groups that you want this user to belong to. Then click *Add to selected groups*."
Or something.